bugsnag-android-core/src/main/java/com/bugsnag/android/EventPayload.kt

```diff
@@ -12,7 +12,7 @@ import java.io.IOException
  */
 class EventPayload @JvmOverloads internal constructor(
     var apiKey: String?,
-    val event: Event? = null,
+    var event: Event? = null,
```

Comment:
Should this really be changed to a `public var`? Possibly we could make the `set` internal?